src/Algebra/Construct/Add/Point.agda

```diff
@@ -0,0 +1,102 @@
+------------------------------------------------------------------------
+-- The Agda standard library
+--
+-- Definition of algebraic structures we get from adding a disjoint point
+------------------------------------------------------------------------
+
+{-# OPTIONS --without-K --safe #-}
+
+module Algebra.Construct.Add.Point where
+
+open import Algebra.Bundles
+open import Algebra.Core using (Op₂)
+import Algebra.Definitions as Definitions
+import Algebra.Structures as Structures
+open import Data.Maybe.Relation.Binary.Pointwise
+open import Data.Product using (_,_)
+open import Level using (Level; _⊔_)
+open import Relation.Binary.Core
+open import Relation.Binary.Definitions
+open import Relation.Binary.Structures
+open import Relation.Nullary.Construct.Add.Point
+
+private
+  variable
+    a ℓ : Level
+
+lift-op : ∀ {A : Set a} → Op₂ A → Op₂ (Pointed A)
+lift-op op [ p ] [ q ] = [ op p q ]
+lift-op _  [ p ] ∙     = [ p ]
+lift-op _  ∙     [ q ] = [ q ]
+lift-op _  ∙     ∙     = ∙
+
+module _ {A : Set a} {_≈_ : Rel A ℓ} {op : Op₂ A} (refl-≈ : Reflexive _≈_) where
+  open Definitions
+  private
+    _≈∙_ = Pointwise _≈_
+
+  lift-≈ : ∀ {x y : A} → x ≈ y → Pointwise _≈_ [ x ] [ y ]
+  lift-≈ = just
+
+  lift-op-cong₂ : Congruent₂ _≈_ op → Congruent₂ _≈∙_ (lift-op op)
```

Comment:
I think we've decided in general that the best way to import and use these `Construct` modules is qualified, e.g.
```
import Algebra.Construct.Add.Point as Point
```
therefore we usually omit the `lift-op` prefix because a) it doesn't add any information, what sort of lifting? what sort of op? and b) it's cleaner to be able to write `Point.cong` rather than `Point.lift-op-cong₂` and c) because you can easily chance the name given to the module whereas you can't rename `lift-op` easily for every proof in the file.

So essentially I think we can drop the `lift-op` in every name.